scss/abstracts/_mixins.scss

```diff
@@ -0,0 +1,40 @@
+@use 'variables' as *;
+
+@mixin flexbox($flex-direction: row, $justify: center, $align: center) {
```

Comment:
I'm not a big fan of creating mixins for flexbox - it's longer than actual flex syntax..

What do you think @Thommyaso @marcinmaciejewski ?